sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc

```diff
@@ -709,6 +709,31 @@ optimize such partial barriers.
 _{endnote}_]
 |====
 
+==== Command Graph
+
+The functions in this section are only available if the
+link:./sycl_ext_oneapi_graph.asciidoc[
+  sycl_ext_oneapi_graph] extension is supported.
+
+|====
+a|
+[frame=all,grid=none]
+!====
+a!
+[source,c++]
+----
+namespace sycl::ext::oneapi::experimental {
+
+void submit(sycl::queue q, command_graph<graph_state::executable> g);
+
```

Comment:
Do we want `submit_with_event()` as well?

Reply:
Good catch. Yes, I think we want both. Will add.